src/DateSelector/index.js

```diff
@@ -71,6 +83,22 @@ class DateSelector extends Component {
     }
   }
 
+  handleSelectedPresets () {
+    const { selectedPreset, presets } = this.props
+
+    if (selectedPreset) {
+      const findEqualKey = ({ key }) => equals(key, selectedPreset)
```

Comment:
This function should include the `find` to be easier to understand.
```suggestion
       const findEqualKey = find(({ key }) => equals(key, selectedPreset))
```
And in the line 94 you will only nedd to call it